tests/integration/conftest.py

```diff
@@ -42,6 +42,10 @@ def find_recipe_folder(base_path: Path, folder_name: str) -> Path:
 def get_task_list(task: str) -> list[OTXTaskType]:
     if task == "all":
         return [task_type for task_type in OTXTaskType if task_type != OTXTaskType.DETECTION_SEMI_SL]
+    if task == "multi_classification":
+        return [OTXTaskType.MULTI_CLASS_CLS, OTXTaskType.MULTI_LABEL_CLS]
```

Comment:
One quick comment is that multi-class-cls will probably work longer than multi-label-cls and h-label-cls combined. It would be nice to separate all three in the future. Feel free to ignore this comment.

Reply:
@harimkang thanks for your comment. actually we can spawn 8 ci-runners on AWS now so split 3 groups of classification will make total test split make to 9. that's the reason why i've divide the classification task into two.